package.json

```diff
@@ -7,8 +7,13 @@
     "test": "cypress run",
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix" 
-},
+    "lint:fix": "npm run lint -- --fix",
+    "clean:reports": "rm -rf ./cypress/reports/*",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "pre": "npm run clean:reports",
+    "post": "npm run combine:reports && npm run generate:reports"
```

Comment:
the pre and post-hooks should have the name of the script as suffix, in which they will be executed after and before. So they should be: `pretest` and `posttest`